`mvn clean install` fails

### Version

5.0.0

### Question

Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)
Maven home: /opt/mvn/apache-maven-3.8.6
Java version: 17.0.6, vendor: Oracle Corporation, runtime: /usr/lib/jvm/jdk-17.0.6
Default locale: en, platform encoding: UTF-8
OS name: "linux", version: "6.5.0-28-generic", arch: "amd64", family: "unix"

last commit: cf3a14008b7cbcf5abc8cf485eec4501566f8469
```
INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  16:45 min
[INFO] Finished at: 2024-05-12T19:22:41+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project jena-integration-tests: Could not resolve dependencies for project org.apache.jena:jena-integration-tests:jar:5.1.0-SNAPSHOT: org.apache.jena:jena-cmds:jar:tests:5.1.0-SNAPSHOT was not found in https://repository.apache.org/snapshots during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of apache.snapshots has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :jena-integration-tests

```
![image](https://github.com/apache/jena/assets/17473366/2f4b8fb2-21a9-47c7-919a-94b723220c70)

--- a/jena-cmds/pom.xml
+++ b/jena-cmds/pom.xml
@@ -180,6 +180,11 @@
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+      </plugin>
+
     </plugins>
   </build>
   

--- a/jena-integration-tests/pom.xml
+++ b/jena-integration-tests/pom.xml
@@ -70,14 +70,6 @@
 
     <dependency>
       <groupId>org.apache.jena</groupId>
-      <artifactId>jena-cmds</artifactId>
-      <version>5.1.0-SNAPSHOT</version>
-      <classifier>tests</classifier>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.jena</groupId>
       <artifactId>jena-geosparql</artifactId>
       <version>5.1.0-SNAPSHOT</version>
     </dependency>

